### PR TITLE
Enable python 3.9

### PR DESCRIPTION
**Pull request by franchuterivera**
This PR enables python 3.9

### CODE REVIEW

**Review by ravinkohli — APPROVED** [review]
Thanks for this PR. I think it can be merged

File: autoPyTorch/pipeline/components/setup/lr_scheduler/CosineAnnealingWarmRestarts.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 class CosineAnnealingWarmRestarts(BaseLRComponent):
     """
     Set the learning rate of each parameter group using a cosine annealing schedule,
-    where \eta_{max}ηmax is set to the initial lr, T_{cur} is the number of epochs
```
Comment (nabenabe0928) [inline autoPyTorch/pipeline/components/setup/lr_scheduler/CosineAnnealingWarmRestarts.py:19]: Why did you delete `\` ?

The usage of the Inline math represetations is nicely described [here](https://numpydoc.readthedocs.io/en/latest/format.html#notes).

```
where :math:`\eta_{\max}` is set to the initial lr, :math:`T_{\text{cur}}` is the number of epochs
since the last restart and :math:`T_{i}` is the number of epochs between two warm
```

**Review by nabenabe0928 — CHANGES_REQUESTED** [review]
Thanks for the PR.
I put a small comment.

File: autoPyTorch/pipeline/components/setup/lr_scheduler/CosineAnnealingWarmRestarts.py (hunk 1 above)
Comment (franchuterivera) [reply autoPyTorch/pipeline/components/setup/lr_scheduler/CosineAnnealingWarmRestarts.py:19]: Hello, sorry this was not an intentional change. 

My vim configuration changed this. It did so because this should be a raw string, as `\e` is not a valid escaped sequence.
